assets/src/components/rightPanel.tsx

```diff
@@ -16,10 +16,12 @@ const RightPanel = ({
 
   // close notification if you move away from ladder page
   // TODO delete when notifications are viewable from anywhere
-  const history = useHistory()
-  if (history) {
+  const location = useLocation()
+  if (location) {
     /* istanbul ignore next */
-    history.listen(() => dispatch(setNotification(undefined)))
+    useEffect(() => {
+      dispatch(setNotification(undefined))
+    }, [location])
```

Comment:
Additionally, if you do end up keeping this new `useEffect` hook, you should make sure to `git rebase master` so that you pull in the ESLint React hooks rule.

Reply:
Since there are conflicts in `package-lock.json`, you'll probably want to accept the changes from `master` and then re-run the `npm install` command(s) you used.